content/en/docs/concepts/overview/components.md

```diff
@@ -3,6 +3,9 @@ reviewers:
 - lavalamp
 title: Kubernetes Components
 content_type: concept
+description: >
+  A Kubernetes cluster consists of a set of worker machines, called nodes , that run containerized applications. Every cluster has at least one worker node.
```

Comment:
nit: Could the text be simplified? The paragraph identifies nodes and control plane but describes them separately.
A cluster needs a control plane component as well as a node?
Here is one way to edit:
A Kubernetes cluster consists of a set of machines called nodes. [is this needed: Every cluster has at least one node.]
The nodes host the Pods that represent the application components and run the containerized applications. The control plane manages the nodes and the Pods in the cluster. In production environments, the control plane usually runs across multiple computers and a cluster usually runs multiple nodes, providing fault-tolerance and high availability.